### ISSUE TITLE
Installation screen - should be pre-populated

### ISSUE THREAD

**Issue by bbusenius** [hahnicity/MEGAndroid#2]
We need to pre-populate as much of the data as possible on the login screen. If fields can't be populated they should be left blank.

![meg-installation-screen-2](https://cloud.githubusercontent.com/assets/7826429/16473628/a159aa96-3e34-11e6-9899-d37e86625ff1.png)

**Comment by JoshuaLyle** [comment]
This has been done in the most recent version

**Comment by bbusenius** [comment]
This didn't happen for me in the APK. It must have come in commit 7b7f1d8fffc75f3cbe8aadae434b070d61fe02ce. I'll install from source tonight.

**Comment by bbusenius** [comment]
I'm going to tentatively reopen this issue. This still isn't working for me.

**Comment by JoshuaLyle** [comment]
I have it to populate with the information you registered with.

I'm assuming this bug is for keeping any data that was input while unregistered and switching between activities?

**Comment by bbusenius** [comment]
Ah no, the idea here is that when the user clicks on the register button, we fill in all the details we can. We wanted this in order to streamline the registration process for novice users. At a minimum their phone number and email would be filled in. I'm not sure if we can get their name but that would be nice. Of course they should be able to change all of this since not everyone will want to use it with their gmail address and people might want to use a different name etc.

**Comment by JoshuaLyle** [comment]
Oh that makes more sense. I'll work on it. What's a scenario where the registration phone number shouldn't be the phone number of that device?

**Comment by bbusenius** [comment]
I don't think that scenario exists. As far as I know the number must always be the same as the device's.
